parenthetics.py

```diff
@@ -0,0 +1,16 @@
+"""parenthetics code-kata."""
+
+
+def parenthetics(string):
+    """Return 1, 0, or -1 depending on the string input."""
+    queue = []
+    for i in string:
```

Comment:
better variable name than `i` is warranted. also a better name than `string`. At least something that isn't so close to a Python built-in